packages/@o3r/rules-engine/builders/rules-engine-extractor/helpers/rules-engine.extractor.ts

```diff
@@ -293,21 +292,23 @@ export class RulesEngineExtractor {
           }
 
           declaration.type.typeArguments?.forEach((argType, idx) => {
```

Comment:
```suggestion
          declaration.type.typeArguments?.slice(0, 2).forEach((argType, idx) => {
```
if we are interested only in the 2 first elements